sdk/ai/Azure.AI.Projects/src/Custom/Datasets/Datasets.cs

```diff
@@ -117,8 +117,9 @@ public ClientResult<FileDatasetVersion> UploadFile(string name, string version,
                 FileDatasetVersion fileDatasetVersion = new FileDatasetVersion(dataUri: dataUri);
                 BinaryContent content = BinaryContent.Create(fileDatasetVersion);
 
-                ClientResult<FileDatasetVersion> result = (ClientResult<FileDatasetVersion>)CreateOrUpdate(name, outputVersion, content);
-                return result;
+                ClientResult result = CreateOrUpdate(name, outputVersion, content);
+
+                return ClientResult.FromValue((FileDatasetVersion)result, result.GetRawResponse());//CreateOrUpdate(name, outputVersion, content);
```

Comment:
Is the comment at the end of the line needed?

Reply:
Fixed